card/mod/05_standard/set/self/all/tasks.rb

```diff
@@ -0,0 +1,17 @@
+
+event :admin_tasks, :initialize, on: :update do
+  return unless (task = Env.params[:task])
+  raise Card::PermissionDenied.new(self) unless Auth.always_ok?
+
+  case task.to_sym
+  when :clear_cache          then Card::Cache.reset_all
+  when :repair_references    then Card::Reference.repair_all
+  when :empty_trash          then Card.empty_trash
+  when :clear_view_cache     then Card::ViewCache.reset
+  when :delete_old_revisions then Card::Action.delete_old
+  when :repair_permissions   then Card.repair_all_permissions
+  when :clear_solid_cache    then Card.clear_solid_cache
+  when :clear_machine_cache  then Card.reset_all_machines
+  end
+  abort success: { card: Card[:stats] }
+end
```

Comment:
I think the rationale was that, eg wth repairing permissions or references, we were conceptually updating all cards (and were not updating the stats card).  Given how our events and subcards have evolved, it seems more normal now to do this as an update on the stats card (an admin card might make even more sense, but we'd have to migrate for that, and I'm not sure it's that important)